src/CMakeLists.txt

```diff
@@ -1358,7 +1358,17 @@ if( NOT "${CMAKE_GENERATOR}" MATCHES "Xcode|Visual Studio*" )
                DESTINATION "${_DATADIR}/mime/packages" )
       install( FILES "${topdir}/presets/EQDefaultCurves.xml"
                DESTINATION "${_PKGDATA}" )
+      install( FILES "${CMAKE_SOURCE_DIR}/linux/audacity"
```

Comment:
Check makefiles generator on macos

Reply:
Should be `install( PROGRAMS` instead of `install( FILES` to get correct permissions on the script. It might be good to call the script `audacity.sh` in the repository to get syntax highlighting and other benefits, then do `RENAME audacity` here.

Ideally this section would work with the Unix Makefiles generator on macOS, but I notice that the lines above install Linux-only files (e.g. `applications/audacity.desktop` and `mime/packages/audacity.xml`) so there's probably not much point in trying to get the shell script work on macOS.